DataManagementSystem/scripts/dirac-admin-allow-se.py

```diff
@@ -88,9 +87,12 @@
   DIRAC.exit(2)
 
 if site:
-  res = gConfig.getOptionsDict('/Resources/Sites/LCG/%s' % site)
+  res = getSites()
   if not res['OK']:
-    gLogger.error('The provided site (%s) is not known.' % site)
+    gLogger.error(res['Message'])
+    DIRAC.exit(-1)
+  if site not in res['Value']:
+    gLogger.error(res['Message'])
```

Comment:
```suggestion
    gLogger.error('The provided site (%s) is not known.' % site)
```
I missed this before, same problem as was in ban-se.